libs/gi/page-team/src/CharacterDisplay/Tabs/TabOptimize/Components/OptimizationTargetEditorList.tsx

```diff
@@ -172,6 +172,9 @@ function OptimizationTargetEditorItem({
         setTarget={(target) => setTarget(target, path, index)}
         defaultText={t('targetSelector.selectBuildTarget')}
         disabled={disabled}
+        targetSelectorModalProps={{
+          excludeSections: ['custom', 'bounsStats', 'character', 'teamBuff'],
```

Comment:
Why no multi opt for min stat?

Reply:
mtargets can have some funky targets inside(since no specific target validation), and they can cause shifts when editing mtarget list (referenced by index), a lot of cornercases for a usecase that I don't really see any nuance in